less/variables.less

```diff
@@ -11,4 +11,42 @@
 @silver: #C0C0C0;
 @shark: #212529;
 @eastern-blue: #17A2B8;
-@sandy-beach: #FFEBCD;
\ No newline at end of file
+@sandy-beach: #FFEBCD;
+
+@mobile: ~"(max-width: 500px)";
+@tablet: ~"(max-width: 850px)";
+
+@keyframes fade-in {
```

Comment:
I think you could consider @keyframes a mixin and put them in the mixin file. That would be appropriate